### DISCUSSION TITLE
Why my account has sinalized with SPAM?

### DISCUSSION THREAD

**Discussion by brunadalmagro** [community/community#62244] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I try to connect into replit and receive this message "You are marked as spam, and therefore cannot authorize a third party application."

**Comment by Piero24** [comment]
@brunadalmagro 
Check your username and email address settings, they often cause problems if they are not configured correctly. Also check if you used an untrusted email when signing up.

Go to your account settings and try changing your username. If you can't do it yourself and are asked to contact support, it means your account is restricted. In this case, contact GitHub Support for the necessary instructions.

**Comment by brunadalmagro** [comment]
thanks

**Reply by bytck** [reply]
Did you find a solution?
